src/plotting/plotSettings.component.tsx

```diff
@@ -373,7 +354,9 @@ const PlotSettings = (props: PlotSettingsProps) => {
                 freeSolo
                 clearOnBlur
                 id="select x axis"
-                options={axisSelectionOptions}
+                options={['timestamp'].concat(
```

Comment:
Not sure if this would ever be a problem - but this `options` array is new on each rerender so would cause the `Autocomplete` to rerender whenever the settings panel changes at all. Would just need to extract it to a `useMemo` if we're concerned about it

Reply:
A good point and another area where there's some inconsistency. Timestamp is not being treated as a channel as such but we still need it to appear in the same place as them in some situations, such as here and as a column in the record table. I'm unclear how to unify all this at the moment

Reply:
Might make it mentally easier keep track to treat timestamp the same as the rest, and just filter it out when we don't need it (e.g. column checkboxes, Y channel selection)